package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/dydx.js",
-  "version": "0.6.4",
+  "version": "0.6.5",
```

Comment:
This might need to be a minor version since there are some breaking changes (from EthWrapperForBucketLender -> BucketLenderProxy)

Reply:
So this is a breaking change, but I think not for the reason Brendan mentioned. The reason it's breaking is because you removed the `beneficiary` argument to `deposit` so anyone trying to call it in the old way will fail.

It doesn't mater from a breaking perspective if a different proxy contract is used behind the scenes as no clients that are using this library will have to update